pkg/controller/statefulset/stateful_pod_control_test.go

```diff
@@ -61,7 +63,7 @@ func TestStatefulPodControlCreatesPods(t *testing.T) {
 	}
 	for i := range events {
 		if !strings.Contains(events[i], v1.EventTypeNormal) {
-			t.Errorf("Expected normal events found %s", events[i])
+			t.Errorf("Found unexpected non-normal event %s", events[i])
```

Comment:
These changes are from a different PR. Looks like they came in by mistake while resolving merge conflicts?